static/src/javascripts/projects/common/modules/commercial/creatives/commercial-component.js

```diff
@@ -103,13 +105,30 @@ define([
         }
     }
 
+    function paidforCardLoaded(el) {
+        var height,
+            $adSlot = $(el);
+
+        if ($adSlot.hasClass('ad-slot--mostpop')) {
```

Comment:
We don't have `js-ad-slot--mostpop`, we only got `js-ad-slot` but it will target all ad slots. In this example I am interested only in the `mostpop` slot.